pages/index.html

```diff
@@ -4,9 +4,98 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <!-- <link rel="stylesheet" type="text/css" href="styles/style.css" /> -->
+  <link href="../styles/style.css" rel="stylesheet">
+  <title>Sana's Website</title>
 </head>
 <body>
+  <header>  
+    <h1>Sana's Swiftie Website</h1>
+
+    <div class="top">
+      <a href="about.html">About ME!</a>
+      <a href="portfolio.html"> My Portfolio</a>
+    </div>
+
+  </header>  
+
+
+<section>
+    <main>
+        <h1>My Favorite Albums</h1> 
+        <div class="grid-container">
+          
+            <div class="grid-item">
+                  <a href="https://taylor.lnk.to/evermorealbumdlxWE">
+                      <img class="cover" alt="Taylor Swift evermore album cover" src="../assets/evermore.png"/>
+                  </a>
+            </div>
+
+            <div>
+              <a href="https://taylor.lnk.to/folkloreWE">
+                <img class="cover" alt="Taylor Swift folklore album cover" src="../assets/folklore.png"/>
+              </a>
+            </div> 
+            
+            <div>
+                <a href="https://taylor.lnk.to/lover">
+                    <img class="cover" alt="Taylor Swift lover album cover" src="../assets/lover.png"/>
+                </a>
+            </div> 
+            
+            <div>
+                <a href="https://taylor.lnk.to/redtaylorsversionWE">
+                    <img class="cover" alt="Taylor Swift red album cover" src="../assets/red.jpeg"/>
+                </a>
+            </div> 
+            
+            <div>
+                <a href="https://taylor.lnk.to/fearlesstaylorsversionWE">
+                    <img class="cover" alt="Taylor Swift fearless album cover" src="../assets/fearless.jpeg"/>
+                </a>
+            </div> 
+
+            <div>
+              <a href="https://taylor.lnk.to/reputation">
+                  <img class="cover" alt="Taylor Swift reputation album cover" src="../assets/reppng.png"/>
+              </a>
+          </div> 
+
+          </div>
+    </main>
+</section>
+
+<section class="contact">
+  <nav>
```

Comment:
Same comment as above, you could remove the section element, put "contact" class on nav, and move nav element into footer